src/components/simple-billboard/index.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Icon } from 'nr1';
+
+/**
+ * @param {Object} metric - metric name, value, previousCalue
+ * @param {string} prefix - prefix for metric.value (i.e. '$')
+ * @param {string} suffix - suffix for metric.value (i.e. users)
+ * @param {string} className -  SCSS class name - gets added to existing JSX classes
+ * @param {Object} style - SCSS style - gets added to JSX - overwrites existing/duplicate CSS properties
+ * @return {JSX Object} - RENDERING name, value, up/down trend when previousValue present
+ */
+const SimpleBillboard = ({ metric, prefix, suffix, className, style }) => {
+  if (metric.value) {
+    metric.value = Number(metric.value);
+  }
+  if (metric.previousValue) {
+    metric.previousValue = Number(metric.previousValue);
+  }
+
+  const MetricStatus = (value) => {
+    let metricStatus = (
+      <Icon
+        type={Icon.TYPE.INTERFACE__CARET__CARET_BOTTOM__WEIGHT_BOLD}
+        color={'red'}
+      />
+    );
+    metricStatus =
+      value === 0 ? (
+        <Icon
+          type={Icon.TYPE.INTERFACE__CARET__CARET_RIGHT__WEIGHT_BOLD}
+          color={'gold'}
+        />
+      ) : (
+        metricStatus
+      );
+    metricStatus =
+      value > 0 ? (
+        <Icon
+          type={Icon.TYPE.INTERFACE__CARET__CARET_TOP__WEIGHT_BOLD}
+          color={'green'}
+        />
+      ) : (
+        metricStatus
+      );
+    return <div className="metric-status">{metricStatus}</div>;
+  };
```

Comment:
Same for this function. Use `useMemo` to have it cached, and you could probably simplify the function with something like this...
```jsx
  const MetricStatus = useMemo(() => {
    let attrs = {
      type: Icon.TYPE.INTERFACE__CARET__CARET_BOTTOM__WEIGHT_BOLD,
      color: 'red',
    };
    if (value === 0) {
      attrs = {
        type: Icon.TYPE.INTERFACE__CARET__CARET_RIGHT__WEIGHT_BOLD,
        color: 'gold',
      };
    } else if (value > 0) {
      attrs = {
        type: Icon.TYPE.INTERFACE__CARET__CARET_TOP__WEIGHT_BOLD,
        color: 'green',
      };
    }
    return (
      <div className="metric-status">
        <Icon {...attrs} />
      </div>
    );
  } , [value]);
```

Reply:
used useCallback()